docs/_data/todos.json

```diff
@@ -1 +1 @@
"file":"elements/pfe-markdown/src/pfe-markdown.js","tag":"TODO","line":64,"ref":"","text":"when we stop supporting IE11, the need to disconnect and"},{"file":"elements/pfe-page-status/src/pfe-page-status.js","tag":"TODO","line":39,"ref":"","text":"Deprecated in 1.0"},{"file":"elements/pfe-primary-detail/src/pfe-primary-detail.js","tag":"TODO","line":32,"ref":"","text":"Add functions to open a specific item by index or ID"},{"file":"elements/pfe-primary-detail/src/pfe-primary-detail.js","tag":"TODO","line":159,"ref":"","text":"decide if we need this anymore"},{"file":"elements/pfe-progress-indicator/src/pfe-progress-indicator.js","tag":"TODO","line":26,"ref":"","text":"Deprecated in 1.0"},{"file":"elements/pfe-progress-steps/src/pfe-progress-steps-item.js","tag":"TODO","line":160,"ref":"","text":"This needs AT"},{"file":"elements/pfe-progress-steps/src/pfe-progress-steps.js","tag":"TODO","line":102,"ref":"","text":"we have to do it in javascript until everyone supports"},{"file":"elements/pfe-select/src/pfe-select.js","tag":"TODO","line":19,"ref":"","text":"Do we need to rename this to remove the prefix?"},{"file":"elements/pfe-tabs/src/pfe-tab-panel.js","tag":"TODO","line":52,"ref":"","text":"Deprecated in 1.0"},{"file":"elements/pfe-tabs/src/pfe-tab.js","tag":"TODO","line":49,"ref":"","text":"Deprecated in 1.0"},{"file":"elements/pfe-tabs/src/pfe-tabs.js","tag":"TODO","line":104,"ref":"","text":"Deprecated for 1.0"},{"file":"elements/pfe-tabs/src/pfe-tabs.js","tag":"TODO","line":110,"ref":"","text":"Deprecated for 1.0"},{"file":"elements/pfe-tabs/src/pfe-tabs.js","tag":"TODO","line":116,"ref":"","text":"Deprecated for 1.0"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":38,"ref":"","text":"the reference to _debugLog is for backwards compatibiilty and will be removed in 2.0"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":60,"ref":"","text":"Add config validation in the future."},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":174,"ref":"","text":"Deprecated with 1.0"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":378,"ref":"","text":"Deprecated for 1.0 release"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":381,"ref":"","text":"Migrate this out of schema for 1.0"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":603,"ref":"","text":"maybe we should use just the attribute instead of the class?"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":634,"ref":"","text":"--theme will be deprecated in 2.0"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":893,"ref":"","text":"add support for a validation function"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":1043,"ref":"","text":"Should this error be reported to the console?"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":1086,"ref":"","text":"add a warning when a component property conflicts with a global property."}]
```

Comment:
@kylebuch8  is it normal for this to appear in the diff like this?

Reply:
It makes sense if this is generates a list of todos.  I've not seen it before though 🤷